test/setup_test.go

```diff
@@ -408,6 +408,17 @@ func testSetup() {
 	})
 }
 
+func getRevisionOfApplication(appName string) string {
```

Comment:
I prefer to use `getApplicationRevision`.

```suggestion
func getApplicationRevision(appName string) string {
```

Reply:
Renamed.